src/components/DocumentationTopic/PrimaryContent/DeclarationGroup.vue

```diff
@@ -80,6 +147,32 @@ export default {
       return this.declaration.platforms.join(', ');
     },
     isSwift: ({ interfaceLanguage }) => interfaceLanguage === Language.swift.key.api,
+    references: ({ store }) => store.state.references,
+    isExpanded: {
+      get: ({ declListExpanded }) => declListExpanded,
+      set(value) {
+        this.$emit('update:declListExpanded', value);
+      },
+    },
+  },
+  methods: {
+    async selectDeclaration(identifier) {
+      if (identifier === this.identifier) return;
+      this.selectedIdentifier = identifier;
+      // enough time to update the just selected item
+      await waitFor(100);
+      this.isExpanded = false; // collapse the list
+      // await animation to finish
+      await waitFor(500);
+      this.$router.push(this.references[identifier].url);
+    },
+    getWrapperComponent(decl) {
+      return (!this.isExpanded || decl.identifier === this.identifier)
+        ? 'div' : 'button';
```

Comment:
Not important, but if this is meant to swap between a button and a div for active vs inactive states, it might be worth exploring if we can just use the disabled attribute on the button instead of changing the element.

Reply:
We could render it as a disabled button instead, ye 🤔

Reply:
I didn't want all the "regular" declaration pills, which are the majority case, to be rendered as a disabled button. What do you think?

Reply:
I think it's fine as-is considering the button is only meant to be used for pages w/ overloaded symbols and this code impacts all pages, regardless of overloads. Was just curious.